Modules/ITS/src/ITSFeeTask.cxx

```diff
@@ -329,7 +329,10 @@ void ITSFeeTask::monitorData(o2::framework::ProcessingContext& ctx)
   resetLanePlotsAndCounters(); // action taken depending on mResetLaneStatus and mResetPayload
 
   for (auto it = parser.begin(), end = parser.end(); it != end; ++it) {
-    auto const* rdh = it.get_if<o2::header::RAWDataHeaderV6>();
+    auto const* rdh = it.get_if<o2::header::RAWDataHeaderV7>();
+    if(!rdh) {
+      ILOG(Debug, Devel) << "We ask RDHv7 but you are running with another version, please fix it";
```

Comment:
```suggestion
      ILOG(Debug, Devel) << "We ask RDHv7 but you are running with another version, please fix it" << ENDM;
```

Reply:
too fast, thanks! Give me few min, I wanted to run a test

Reply:
@iravasen One can have it in a version-independent way as:
```
const auto& rdh = *reinterpret_cast<const header::RDHAny*>(it.raw());
auto feeID = o2::raw::RDHUtils::getFEEID(rdh);
...
```
See other getters of https://github.com/AliceO2Group/AliceO2/blob/90a2345c0c322dafcfa6c33b2fa2a341649ff754/Detectors/Raw/include/DetectorsRaw/RDHUtils.h#L36